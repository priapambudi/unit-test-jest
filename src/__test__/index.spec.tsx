import React from "react";
import { render } from "@testing-library/react";
import HomepageView from "../components/pages/Home";

describe("render home", () => {
  it("should render home", () => {
    const { container } = render(<HomepageView />);
    expect(container).toMatchSnapshot();
  });
});
