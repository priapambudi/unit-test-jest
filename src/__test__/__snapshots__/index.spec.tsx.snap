// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render home should render home 1`] = `
<div>
  <div>
    Homepage
  </div>
</div>
`;
